lib/start_with_right_hostname.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+
+if [ ! -d "/opt/ml/config" ]
```

Comment:
See line https://github.com/aws/sagemaker-pytorch-container/pull/123/files#diff-f4c212a709d5a6c846dcd140670409fbL5 , the correct folder should be /opt/ml/input/config

Reply:
fixed